marlowe-playground-client/src/Marlowe/ViewPartials.purs

```diff
@@ -0,0 +1,114 @@
+module Marlowe.ViewPartials where
```

Comment:
I moved this from the Static analysis bottom panel to this place because there was a cyclic dependency. But I'm not 100% sold on the module name 🤔

Reply:
It seems it is all about displaying warnings for now, how about `ViewWarnings`?

Reply:
Mhmmh, could be. I've called like this because I imagine later on it could be useful to have other Marlowe related templates/partials/views